polynote-frontend/polynote/ui.js

```diff
@@ -710,9 +710,10 @@ export class NotebookUI extends UIEventTarget {
         this.editBuffer = new EditBuffer();
 
         this.cellUI.addEventListener('UpdatedConfig', evt => {
+            console.log("updating config")
```

Comment:
We should do a purge of `console.log`s before alpha

Reply:
Woops, I thought I'd gotten all of 'em but I guess I missed that one!